### ISSUE TITLE
Dicebot integration with Discord and/or Roll20

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#3764]
TODO: 
- [ ] Pester Carlton on Discord about an API for Exploding Dice.
- [ ] Use diagnostics on the Dicepool form to make it output to Discord.
- [ ] Continue to corner the market on chargen options

**Comment by BKlaassen** [comment]
For roll20 integration it might be an idea to output the corresponding command that would need to be entered

**Comment by chummer5a** [comment]
Yeah, that'd basically be how it would work. Dicebot button would calculate the pool for you, either API key to push it to chat or copy paste.
